src/Network.cpp

```diff
@@ -419,12 +422,12 @@ Network::Netresult Network::get_scored_moves_internal(
     std::vector<float> winrate_data(256);
     std::vector<float> winrate_out(1);
     // Data layout is input_data[(c * height + h) * width + w]
-    input_data.reserve(INPUT_CHANNELS * width * height);
-    for (int c = 0; c < INPUT_CHANNELS; ++c) {
+    net_t* input_ptr = input_data.data();
```

Comment:
No need to revert to raw pointer use here. Yes, you can't use emplace_back (or push_back) any more but that is because the array is already initialized, which is actually not what you want.